ctapipe/coordinates/camera_frame.py

```diff
@@ -19,35 +19,98 @@
 from .representation import PlanarRepresentation
 
 
+def barrel_distortion(x, y, alpha=0):
```

Comment:
If my reading is correct, this implements barrel distortion for alpha > 0 and pincushion distortion for alpha < 0

Reply:
Also: alpha * r is not unit less. Should it be something like `alpha * r / focal_length`?